### ISSUE TITLE
This repos is against the FotMob TOS

### ISSUE THREAD

**Issue by cnordvik** [Abdo30004/Fotmob-Api#1]
https://fotmob.com/tos.txt

**Comment by Abdo30004** [comment]
Hello , is this an official request to take down the repository?

**Comment by cnordvik** [comment]
Yes please. Thank you for understanding! Any questions please reach out to legal@fotmob.com.

**Comment by Abdo30004** [comment]
Well , since it is a take down request, i prefer to receive it in my email yahiaoui.abderrahmane.pro@gmail.com. The repository isn't the only available source code , there is also an npm package.
